### PR TITLE
Typography/Korean lang no word break

### PR DESCRIPTION
**Pull request by somethiiing**
add word-break: keep-all for korean

### CODE REVIEW

**Review by zprong — APPROVED** [review]
I see this would also be the case for Chinese and Japanese -- can we update this to include `ja` and `zh`?

**Review by fransiskusx — APPROVED** [review]
lgtm

**Comment by zprong** [comment]
source: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#keep-all

**Comment by somethiiing** [comment]
> source: https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#keep-all

for zh and ja, i think we need a different strategy. it doenst work entirely as intended when i tested it

File: packages/syntax-core/src/Typography/Typography.module.css [hunk 1]
```diff
@@ -229,3 +229,8 @@
 .preLine {
   white-space: pre-line;
 }
+
+:lang(ko),
+.typography {
```
Comment (christianvuerings) [inline packages/syntax-core/src/Typography/Typography.module.css:234]: @somethiiing Did you check whether this is correct? I noticed that all text now has `word-break: keep-all`, not just Korean.

```suggestion
/* stylelint-disable-next-line selector-max-compound-selectors -- https://github.com/w3c/csswg-drafts/issues/4285 */
:lang(ko) .typography {
  word-break: keep-all;
}
```

Also, I would move these lines to where we have the other language specific overrides:
https://github.com/Cambly/syntax/blob/76d59226915367c84f9ccdf0497de9911c146398/packages/syntax-core/src/Typography/Typography.module.css#L57-L58

image

**Pull request merged**